bin/node-template/node/src/service.rs

```diff
@@ -100,6 +101,8 @@ pub fn new_partial(
 
 	let (grandpa_block_import, grandpa_link) = sc_consensus_grandpa::block_import(
 		client.clone(),
+		// false positive
+		#[allow(clippy::redundant_clone)]
```

Comment:
Should be solved properly.

Reply:
This comment (+ allow annotation) is to record that we run into a clippy issue here. If accepted, I can then easily open a clippy issue pointed at this comment. Once it is fixed on the clippy side we can get rid of this comment and annotation.